### ISSUE TITLE
Weather Info Source

### ISSUE THREAD

**Issue by davidearlyoung** [null8626/python-weather#57]
Hello,

This is just a friendly suggestion based on my experience. I want to suggest that it would be smart to let people know where you are sourcing weather details for this project to work.

I was frustrated in having to dig into your code in order to find out that your using `wttr.in`. Which in turn uses WMO (World Meteorological Organization) and NWS (National Weather Service) from what I'm hearing.

Reason this is important is that I live in a place where it's harder to predict accurate weather. And different weather organizations do better with their predictions then others. Not having this detail upfront would be one strong reason for me to walk away from using your project.

Adjustments could be something simple as putting a line or two in the main README for this projects repo.

I hope that this helps. Thanks for taking time to read this.

**Comment by null8626** [comment]
Hai! Thank you so much for this. Will do!

**Comment by null8626** [comment]
After looking at `wttr.in`'s source, it seems that the project uses [World Weather Online](https://www.worldweatheronline.com/weather-api/) though, as noted from [this function](https://github.com/chubin/wttr.in/blob/575908ae51010b2d71c0b1d4bf4def997bea0fa3/lib/weather_data.py#L9).

**Comment by davidearlyoung** [comment]
Ah, Interesting. I saw something else in a different file. I'll see if I can dig that up in case that is useful for you. Thanks for looking into this.

**Comment by davidearlyoung** [comment]
Also, just as a side note, here is where I found that your lib used wttr.in:

See: `python-weather/python_weather/client.py` (line 87).

https://github.com/null8626/python-weather/blob/3b5f8fe31733643fe840b37159d4a8e43a043d57/python_weather/client.py#L87)

Which I believe is your current for your main branch for that file.

I'm curious now where I saw that it was WMO and not WWO for my source of info. If I find it and it seems useful or helpful, I likely share that info.
